ui/pages/confirmations/confirm/confirm.tsx

```diff
@@ -25,12 +24,16 @@ const Confirm = () => {
   syncConfirmPath();
   setConfirmationAlerts();
   const processAction = useConfirmationAlertActions();
+  const [showAdvancedDetails, setShowAdvancedDetails] = React.useState(false);
```

Comment:
Minor, but do we usually import the hooks directly so `useState` rather than `React.useState`?